pkg/graphviz/traceflow.go

```diff
@@ -475,6 +484,64 @@ func GenGraph(tf *crdv1alpha1.Traceflow) (string, error) {
 		}
 		edge.Attrs[gographviz.Constraint] = "false"
 	}
-
+	if tf.Spec.LiveTraffic && tf.Status.Phase == crdv1alpha1.Succeeded {
+		err = createCapturedPacketNode(graph, cluster2.Name, tf)
+		if err != nil {
+			return "", err
+		}
+	}
 	return genOutput(graph, false), nil
 }
+
+func getCapturedPacketLabel(tf *crdv1alpha1.Traceflow) string {
+	label := "Captured Packet:\\lDestination IP:" + tf.Status.CapturedPacket.DstIP + "\\l" +
+		"Source IP: " + tf.Status.CapturedPacket.SrcIP + "\\l" +
```

Comment:
Sorry, probably put Source IP first, before Destination IP, to be consistent with the graph.

Reply:
@jianjuns  I just updated the order, I was following the output of  capturedpacket in the traceflow result before,  so I am wondering if it's necessary to reorder the sequence in the output as well?

Reply:
Sorry, do you mean antctl traceflow results? I think for that we are printing the yaml struct. You think how could we reorder the fields?

Reply:
Sorry, do you mean antctl traceflow results? I think for that we are printing the yaml struct. You think how could we reorder the fields?

Reply:
ah, OK, got it.